tests/commands/run_test.py

```diff
@@ -81,7 +82,7 @@ def _test_run(repo, options, expected_outputs, expected_ret, stage):
         stage_a_file()
     args = _get_opts(**options)
     ret, printed = _do_run(repo, args)
-    assert ret == expected_ret
+    assert ret == expected_ret, printed
```

Comment:
I'd imagine:

`assert ret == expected_ret, (ret, expected_ret, printed)`

would be more useful